docker/osrdyne-pr-tests.yml

```
@@ -0,0 +1,16 @@
+worker_driver:
+  type: "DockerDriver"
+  network: "osrd_pr_test"
+  container_prefix: "osrd"
```

Change the prefix so it doesn't conflict. Maybe `pr-osrd` ?

```suggestion
  container_prefix: "pr-osrd"
```